src/npc_arena1v1.cpp

```diff
@@ -279,8 +279,12 @@ class npc_1v1arena : public CreatureScript
 
         case 3: // Leave Queue
         {
+            uint8 arenaType = ARENA_TYPE_5v5;
+            if (player->InBattlegroundQueueForBattlegroundQueueType(BATTLEGROUND_QUEUE_5v5))
+                arenaType = ARENA_TYPE_5v5;
```

Comment:
why this reassignment to the same value?

Reply:
> This PR contain code duplicate of #4
> 
> ![immagine](https://user-images.githubusercontent.com/519778/62780200-6eaa1d00-bab5-11e9-917d-0d3d1c72fcf5.png)
> 
> I think we could close it and focus on #4 that contains more improvements, do you agree?
> @poszer, @Winfidonarleyan?

Are you serious? :D